rust/log/src/grpc_log.rs

```diff
@@ -212,14 +219,44 @@ impl Configurable<GrpcLogConfig> for GrpcLog {
                 Ok(client)
             };
         let client = client_for_conn_str(connection_string)?;
-        let alt_client = if let Some(alt_host) = my_config.alt_host.as_ref() {
-            let connection_string = format!("http://{}:{}", alt_host, port);
-            tracing::info!("connecting to alt host {connection_string}");
-            Some(client_for_conn_str(connection_string)?)
-        } else {
-            None
+
+        // Alt client manager setup
```

Comment:
This could be made a bit cleaner, will leave that for another pr since the other use already did it this way